tools/serverpod_cli/test/database/create_definition/database_action_definition_test.dart

```diff
@@ -2,11 +2,13 @@ import 'package:serverpod_cli/analyzer.dart';
 import 'package:serverpod_cli/src/database/create_definition.dart';
 import 'package:serverpod_cli/src/test_util/builders/class_definition_builder.dart';
 import 'package:serverpod_cli/src/test_util/builders/foreign_relation_definition_builder.dart';
+import 'package:serverpod_cli/src/test_util/builders/generator_config_builder.dart';
 import 'package:serverpod_cli/src/test_util/builders/serializable_entity_field_definition_builder.dart';
 import 'package:serverpod_service_client/serverpod_service_client.dart';
 import 'package:test/test.dart';
 
 void main() {
+  var config = GeneratorConfigBuilder().build();
```

Comment:
Suggestion: Remove the GeneratorConfigBuilder and just define the name and modules directly in the test.

Since we don't really need the object anymore it is more explicit if we create the things we need for the test.